content/en/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: blog
+title: "Kubernetes 1.18: Fit & Finish"
```

Comment:
```suggestion
title: 'Kubernetes 1.18: Fit & Finish'
```

Reply:
There are issues in Netlify builds around double quotes and colon characters, but we've found single quotes fix that issue!